src/TorchVision/Utils.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using TorchSharp;
+using static TorchSharp.torch;
+using static TorchSharp.torchvision.io;
+
+namespace TorchSharp
+{
+    public static partial class torchvision
+    {
+        public static class utils
+        {
+            public static Tensor make_grid(
+                IEnumerable<Tensor> tensors,
+                long nrow = 8,
+                int padding = 2,
```

Comment:
@NiklasGustafsson Do you prefer 'long' for most int parameters, and 'double' for most float parameters. I am going by the pytorch type hints.

Reply:
I have to confess that the code is not entirely consistent. Double, yes, but the integer size hasn't been very consistent.